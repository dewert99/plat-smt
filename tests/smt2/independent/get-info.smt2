(get-info :name)
(get-info :authors)
(get-info :error-behavior)
(get-info :assertion-stack-levels)
(push)
(get-info :assertion-stack-levels)
(pop)
(get-info :assertion-stack-levels)