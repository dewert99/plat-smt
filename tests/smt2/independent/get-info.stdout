(:name "PlatSmt")
(:authors "David Ewert")
(:error-behavior continued-execution)
(:assertion-stack-levels 0)
(:assertion-stack-levels 1)
(:assertion-stack-levels 0)
